.github/workflows/pull_request.yml

```diff
@@ -17,8 +21,21 @@ jobs:
         node-version: 14.17.0
         registry-url: 'https://registry.npmjs.org'
     - name: Install dependencies
-      run: yarn install --immutable
+      run: |
+        npm install -g yarn
```

Comment:
I global install necessary?

Reply:
Otherwise lgtm

Reply:
I think it's ok, because it runs only in docker image on github

Reply:
I'd also opt for removing this and only add this if it breaks (we're other github actions that also depend on yarn and don't need this install).